e2etest/bgp_test.go

```diff
@@ -111,17 +130,56 @@ func setupContainers(ipv4Addresses, ipv6Addresses []string) []*frrcontainer.FRR
 
 		ibgpContainerConfig.IPv4Address = ipv4Addresses[0]
 		ibgpContainerConfig.IPv6Address = ipv6Addresses[0]
-		ebgpContainerConfig.IPv4Address = ipv4Addresses[1]
-		ebgpContainerConfig.IPv6Address = ipv6Addresses[1]
-		res, err = createFRRContainers(ibgpContainerConfig, ebgpContainerConfig)
+		multiHopGWContainerConfig.IPv4Address = ipv4Addresses[1]
+		multiHopGWContainerConfig.IPv6Address = ipv6Addresses[1]
+
+		var out string
+		out, err = executor.Host.Exec(executor.ContainerRuntime, "network", "create", multiHopNetwork, "--ipv6",
+			"--driver=bridge", "--subnet=172.30.0.0/16", "--subnet=fc00:f853:ccd:e798::/64")
+		if err != nil {
+			return res, errors.Wrapf(err, "failed to create %s: %s", multiHopNetwork, out)
+		}
+
+		res, err = createFRRContainers(ibgpContainerConfig, ebgpContainerConfig, multiHopGWContainerConfig)
```

Comment:
By doing this, we are using the `multi-hop-router` as ebgp container, which makes sense, but it's not clear from the name. I'd rather call the containers as ebgpSingleHop / ebgpMultiHop and add a comment about how we wire them up.
All the wiring is localized, so it will be easier to follow.

Reply:
Changed the names and added a comment explaining what we're doing and when.